misk-redis/src/main/kotlin/misk/redis/RedisModule.kt

```diff
@@ -0,0 +1,14 @@
+package misk.redis
+
+import misk.inject.KAbstractModule
+import redis.clients.jedis.Jedis
+
+class RedisModule(val config: RedisConfig): KAbstractModule() {
+  override fun configure() {
+    val jedis = Jedis(config.host_name, config.port)
```

Comment:
does this actually attempt to bind to the redis? If so you'll want to move it out to a `Service`

Reply:
either way, you'll eventually want a `Service` to manage the lifecycle connection to the `Redis`

Reply:
Added a `RedisService` to manage the connection lifecycle for redis